slng.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+"""
+    Runs as a command line script
+"""
+import sys
+import requests
+
+
+def get_word(word):
+    """
+        Searches for word and gets a response from urbandictionary API
+    """
+    req = requests.get(
+        'http://api.urbandictionary.com/v0/define?', {'term': word})
+    return req.json()['list']
+
+
+def print_definitions(word):
+    """
+        Loops through first 3 results and prints them out to console
+    """
+    definitions = get_word(word)
+    for i in range(0, 3):
```

Comment:
We should normalize `word`. Maybe for each instead of explicit for to account for words that have fewer than 3 definitions.

Reply:
May need to add logic to return "up to" three definitions